v2/auth_test.go

```diff
@@ -50,7 +50,7 @@ func TestBearerAuth(t *testing.T) {
 	}
 
 	for _, tc := range cases {
-		client := newTestClient(t, tc.name, Version2_11(), true, httpChecks{}, httpReaction{})
+		client := newTestClient(t, tc.name, Version2_11(), true, "", httpChecks{}, httpReaction{})
```

Comment:
I think it's very useful, when you pass an empty string or a nil to a method, to put a comment inline like so explaining the value:

```go
doIt("" /* frobulator */)
```